cmd/ocm-backplane/accessrequest/getAccessRequest.go

```diff
@@ -0,0 +1,109 @@
+package accessrequest
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	ocmcli "github.com/openshift-online/ocm-cli/pkg/ocm"
```

Comment:
You may not need to import ocm-cli here , it's already available on ocm client 
https://github.com/openshift/backplane-cli/blob/4ce37a3a4a21e8b11f16d1ec69d9103239d0fe12/pkg/ocm/ocmWrapper.go#L9

Reply:
Not sure to understand you.
This package is imported on the following files:
```
> grep -rFl github.com/openshift-online/ocm-cli/pkg/ocm . | grep -vF _test | grep -vFi mock
./cmd/ocm-backplane/accessrequest/getAccessRequest.go
./cmd/ocm-backplane/accessrequest/createAccessRequest.go
./cmd/ocm-backplane/cloud/credentials.go
./cmd/ocm-backplane/cloud/console.go
./ocm-backplane
./pkg/ocm/ocmWrapper.go
```
File in imported in the commands which wish to perform several OCM call in the row with the same OCM client because, I guess, it is taking time to re-create the client.

In the new `ocm backplane accessrequest create` command, this is exactly what we are doing:
1. We first check the existence of an existing access request
2. If there is none, we create one.

Both OCM calls will use the same client. The first call will be done through the `accessrequest.getAccessRequest` method which needs the OCM client as a parameter.

For the sake of factorisation, the same method is called in the `ocm backplane accessrequest get` handled by this file.  
Hence the OCM client needs to be created prior calling that method by calling `ocmcli.NewConnection().Build()`, hence the need to import that package.

Reply:
My thinking was to encapsulate the `ocm.NewConnection().Build()` into the `ocmWrapper` interface, where all ocm connections and implementation go to one module. 
If there are multiple call issues, we can improve the `ocmWrapper` interface implementation by caching the connection. 
anyway, you can decide what is the best approach for this scenario

Reply:
Indeed, the `DefaultOCMInterfaceImpl` could manage the `*ocmsdk.Connection` itself.
The only problem would be: when to `Close()` the connection.
Answer probably being: when the `ocm-backplane` CLI exits.
Remark that not properly closing the connection is probably not that of a big a deal. So ideally we should:
- Add a `Close()` method to `OCMInterface`. In `DefaultOCMInterfaceImpl`, this would close the underlying connection if there is one.
- Add a `defer OCMInterface.Close()` in the main method.
WDYT? Could this go in a separate PR? I already want to cleanup a bit the code used to manage the config :)

Reply:
Thanks @Nikokolas3270 for the suggestion, let's add `defer OCMInterface.Close()` into the main method but will do it in separate PR